src/resources/ble_esp32.cc

```diff
@@ -233,6 +233,21 @@ class BleReadWriteElement : public BleErrorCapableResource {
     return unvoid_cast<BleReadWriteElement*>(arg)->_on_access(ctxt);
   }
 
+  static uint16_t mbuf_total_len(os_mbuf* om) {
+    uint16_t total_len = om->om_len;
+    om = SLIST_NEXT(om, om_next);
+    while (om) {
+      total_len += om->om_len;
+      om = SLIST_NEXT(om, om_next);
+    }
+
+    return total_len;
+  }
+
+  uint16_t send_mbuf_len(void) {
```

Comment:
Maybe even consider making this `const`?

```suggestion
  int send_mbuf_len() {
```

Reply:
Maybe just remove this function and inline the call later? At least rename it to mbuf_to_send_len